homeassistant/components/deako/light.py

```diff
@@ -0,0 +1,107 @@
+"""Binary sensor platform for integration_blueprint."""
+import logging
+from typing import Any
+
+from pydeako.deako import Deako
+
+from homeassistant.components.light import ATTR_BRIGHTNESS, ColorMode, LightEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+
+# Model names
+MODEL_SMART = "smart"
+MODEL_DIMMER = "dimmer"
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config: ConfigEntry,
+    add_entities: AddEntitiesCallback,
+) -> None:
+    """Configure the platform."""
+    client: Deako = hass.data[DOMAIN][config.entry_id]
+
+    devices = client.get_devices()
+    if len(devices) == 0:
+        # If deako devices are advertising on mdns, we should be able to get at least one device
+        _LOGGER.warning("No devices found from local integration")
+        await client.disconnect()
+        return
+    lights = [DeakoLightEntity(client, uuid) for uuid in devices]
+    add_entities(lights)
+
+
+class DeakoLightEntity(LightEntity):
+    """Deako LightEntity class."""
+
+    # retype because these will be set
+    _attr_unique_id: str
+    _attr_supported_color_modes: set[ColorMode]
+
+    _attr_has_entity_name = True
+    _attr_name = None
+    _attr_is_on = False
```

Comment:
Should this be False? I think the default is `None` which is unknown which looks like a more valid state